plugins/meson.build

```diff
@@ -18,7 +18,7 @@ if get_option('plugin_amt')
 subdir('amt')
 endif
 
-if get_option('plugin_thunderbolt') and udev.found()
+if get_option('plugin_thunderbolt')
```

Comment:
You should split this into a separate commit explaining that udev isn't in the build env but a runtime dep etc.

Reply:
Right, I'll push that chunk right to master. Thanks.